src/hotspot/share/gc/shared/ageTable.cpp

```diff
@@ -105,25 +106,30 @@ uint AgeTable::compute_tenuring_threshold(size_t desired_survivor_size) {
 }
 
 void AgeTable::print_age_table(uint tenuring_threshold) {
-  if (log_is_enabled(Trace, gc, age) || UsePerfData || AgeTableTracer::is_tenuring_distribution_event_enabled()) {
-    log_trace(gc, age)("Age table with threshold %u (max threshold " UINTX_FORMAT ")",
-                       tenuring_threshold, MaxTenuringThreshold);
-
-    size_t total = 0;
-    uint age = 1;
-    while (age < table_size) {
-      size_t wordSize = sizes[age];
-      total += wordSize;
-      if (wordSize > 0) {
-        log_trace(gc, age)("- age %3u: " SIZE_FORMAT_W(10) " bytes, " SIZE_FORMAT_W(10) " total",
-                            age, wordSize * oopSize, total * oopSize);
-      }
-      AgeTableTracer::send_tenuring_distribution_event(age, wordSize * oopSize);
-      if (UsePerfData) {
-        _perf_sizes[age]->set_value(wordSize * oopSize);
-      }
-      age++;
-    }
+  LogTarget(Trace, gc, age) lt;
+  if (lt.is_enabled() || _use_perf_data || AgeTableTracer::is_tenuring_distribution_event_enabled()) {
+    LogStream st(lt);
+    print_on(&st, tenuring_threshold);
   }
 }
 
+void AgeTable::print_on(outputStream* st, uint tenuring_threshold) {
+  st->print_cr("Age table with threshold %u (max threshold " UINTX_FORMAT ")",
+           tenuring_threshold, MaxTenuringThreshold);
```

Comment:
```suggestion
  st->print_cr("Age table with threshold %u (max threshold " UINTX_FORMAT ")",
               tenuring_threshold, MaxTenuringThreshold);
```
arguments should be aligned after the bracket above